docs/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## Next
+* Added a toggle button for sorting tabs in the records page.
```

Comment:
I'm not sure what "tabs" refers to here?

Reply:
I referring to what I'm pretty sure are the presentation nodes on the page.